vtysh/bgp_vty.c

```diff
@@ -2694,6 +2694,20 @@ bgp_neighbor_peer_group_insert_to_bgp_router(const struct ovsrec_bgp_router *
     free(bgp_neighbor_peer_group_list);
 }
 
+static bool
+is_ibgp_peer(const struct ovsrec_bgp_neighbor *ovs_bgp_neighbor,
```

Comment:
Do you mean changing parameters in the function or function name?

Tried to change parameters in following way:
```
static bool is_ibgp_peer(const struct ovsrec_bgp_neighbor *ovs_bgp_neighbor,
                                      const struct ovsrec_bgp_router *bgp_router_context)
```

But, as I understand, struct ovsrec_bgp_router does not contain asn, it is stored in `ovs_vrf->key_bgp_routers[j]`
E.g., in vtysh_router_context_bgp_clientcallback function https://github.com/MERAprojects/ops-cli/blob/meraswitch/devel/master/vtysh/vtysh_ovsdb_router_context.c

I can write a comment describing parameters in is_ibgp_peer function.